mqtt-cli-plugins/src/main/kotlin/com/hivemq/cli/native_image/tasks/DownloadGraalJVMTask.kt

```diff
@@ -96,6 +96,8 @@ abstract class DownloadGraalJVMTask @Inject constructor(
             "amd64"
         } else if (DefaultNativePlatform.getCurrentArchitecture().isArm) {
             "aarch64"
+        } else if (DefaultNativePlatform.getCurrentArchitecture().name == "arm-v8") {
+            "aarch64"
```

Comment:
```suggestion
        } else if (DefaultNativePlatform.getCurrentArchitecture().isArm || DefaultNativePlatform.getCurrentArchitecture().name == "arm-v8") {
            "aarch64"
```

Reply:
I would rather prefer to keep the if statements separated as they belong to separate architectures.